modules/terraform/cmd.go

```diff
@@ -17,6 +17,17 @@ func GetCommonOptions(options *Options, args ...string) (*Options, []string) {
 		args = append(args, "-no-color")
 	}
 
+	// If Lock is set to true, override the lock argument that has been set (ignore false)
+	if options.Lock == true {
+		lockKey := "-lock"
+		for i, argValue := range args {
+			parts := strings.Split(argValue, "=")
+			if parts[0] == lockKey {
+				args[i] = fmt.Sprintf("%s=true", lockKey)
```

Comment:
Oh, also, if `options.Lock` is `false`, shouldn't this code still run, but set it to false?

Reply:
That is a very good point. In the plan command, -lock=true is currently set, so it would be good if the false was supported here too: https://github.com/gruntwork-io/terratest/blob/master/modules/terraform/plan.go#L36